app/scripts/views/ready.js

```diff
@@ -57,6 +58,29 @@ function (Cocktail, BaseView, Template, Session, Xss, Strings,
       return this._createMarketingSnippet();
     },
 
+    /**
+     * Creates the service name for the ready template
+     * Can either be a string or a link to the relier redirectUri
+     *
+     * @returns {String} serviceName
+     * @private
+     */
+    _createServiceName: function () {
+      var relier = this.relier;
+      var serviceName = relier.get('serviceName');
+      var redirectUri = relier.get('redirectUri');
+      var autoRedirect = relier.get('autoRedirect');
+
+      // if this relier wants to always redirect the user then create a link to the service
+      if (autoRedirect && redirectUri && Url.isNavigable(redirectUri)) {
+        serviceName = Strings.interpolate('<a href="%s" class="no-underline" id="redirectTo">%s</a>', [
+          Xss.href(redirectUri), serviceName
+        ]);
```

Comment:
> Given #2349 where Hello wants to use a different URL entirely for this link

Yeah that one could be something like "Learn more about Hello" instead of "Proceed". Makes it different from what we are trying to do here.